etc/notebooks/got_scotch_demo/scotch_dashboard_declarative.ipynb

```diff
@@ -0,0 +1,783 @@
+    "collapsed": true,
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "We will drop the cluster column from *features\\_df*. We don't need it here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "urth": {
+     "dashboard": {}
+    }
+   },
+   "outputs": [],
+   "source": [
+    "features_df = features_df.drop('cluster', axis=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "## Define Functions <span style=\"float: right; font-size: 0.5em\"><a href=\"#Got-Scotch?\">Top</a></span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "Now we need a couple of functions to retrieve similar scotches and get the feature table of a scotch.\n",
+    "\n",
+    "First, let's define a *get_similar* function, essentially a copy from the [scotch_dashboard notebook](./scotch_dashboard.ipynb) to return the data of the top n similar scotches to a given scotch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "urth": {
+     "dashboard": {}
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def get_similar(name, n, top=True):\n",
+    "    a = sim_df[name].order(ascending=False)\n",
+    "    a.name = 'Similarity'\n",
+    "    df = pd.DataFrame(a) #.join(features_df).iloc[start:end]\n",
+    "    return df.head(n) if top else df.tail(n)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "But since urth-viz-table needs the index (distillery) as a regular column with the data, we need to add that column."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "urth": {
+     "dashboard": {}
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def get_similar_for_table(name, n=6):\n",
+    "    # select n similar scotches except the first which is the same as name\n",
+    "    simTable = get_similar(name, n).iloc[1:]\n",
+    "    # turn the index (distillery) into a regular column\n",
+    "    return simTable.reset_index()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "We then need another function to retrieve the feature data for a set of scotches as *feature_keys*.\n",
+    "\n",
+    "And because urth-viz-* requires the data series as columns as opposed to rows, we will have to transpose this dataframe.\n",
+    "\n",
+    "Then similarly to the similar table above, we also need to add the index (feature) as a regular column."
```

Comment:
Same thing here: not adding a column but rather resetting the index as a column.